package.json

```diff
@@ -17,7 +17,12 @@
   },
   "keywords": [
     "eslint",
-    "config"
+    "eslintconfig",
+    "config",
+    "brigade",
+    "javascript",
+    "react",
+    "stylesheet"
```

Comment:
Ah. I was copying what Airbnb has heh. It is _kind of_ a stylesheet though... happy to remove if you want, I have no opinions about it